exporter.js

```diff
@@ -73,7 +73,7 @@ const createBody = (meta) => {
     }` +
     `\nHash Digest: ${meta.digest}` +
     `\nTest URLs: ${meta.urls.join(', ')}` +
-    (meta.version == 'Dev' ?
+    (meta.version.startsWith('Dev') ?
```

Comment:
```suggestion
    (meta.version.includes('-') ?
```

Reply:
I'm skeptical about using a hyphen to indicate a dev version or not, but I'm down for using `includes()` instead!